packages/web/src/store/playlist-library/helpers.test.js

```diff
@@ -443,3 +445,147 @@ describe('containsTempPlaylist', () => {
     expect(ret).toEqual(false)
   })
 })
+
+describe('addFolderToLibrary', () => {
+  it('Adds a new folder to the end of a playlist library and returns the result', () => {
+    const library = {
+      contents: [
+        { type: 'playlist', playlist_id: 1 },
+        { type: 'playlist', playlist_id: 2 },
+        { type: 'playlist', playlist_id: 3 },
+        { type: 'temp_playlist', playlist_id: 'asdf' }
+      ]
+    }
+    const folder = {
+      id: 'fake-uuid',
+      name: 'Foldero',
+      contents: [],
+      type: 'folder'
+    }
+    const ret = addFolderToLibrary(library, folder)
```

Comment:
personally like using semantic variable names where possible! im guessing ret was used in the other tests, but yeah wouldnt mind using something like `result`?